swagger-v1/swagger.yaml

```diff
@@ -59,6 +59,16 @@ paths:
           type: string
           format: uuid
           example: 3f326510-8fb0-4565-9de1-a090aaa49ac2
+      - name: searchPhrase
+        in: query
+        description: |
+          String to match Merchant Name or Id. Can be entire name/ Id or a partial match.
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: string
+          example: Fine Woodworking
```

Comment:
Think since this is a query string parameter the example might need to be Fine+Woodworking. Probably want to confirm that though.

Reply:
tested and changed example. Thank you!